CONTRIBUTING.md

```diff
@@ -45,6 +45,8 @@ You'll always get credit for your work.
 
     ```bash
     $ pip install --editable .[dev]
+    # zsh workaround
+    $ pip install --editable ".[dev]"
```

Comment:
This should work on both bash and zsh. Rather than having two separate lines I'd much rather have one that works in both shells. Could you remove the other line?

Reply:
Good call! Just updated.